test/integration/env/test_env.py

```diff
@@ -0,0 +1,33 @@
+import os
+import re
+import subprocess
+
+from utils import config_name
+
+
+def test_env_output(tt_cmd, tmpdir):
+    configPath = os.path.join(tmpdir, config_name)
+    # Create test config
+    with open(configPath, 'w') as f:
+        f.write('env:\n  bin_dir:\n  inc_dir:\n')
+    binDir = str(tmpdir + "/bin")
+    tarantoolDir = "TARANTOOL_DIR=" + str(tmpdir + "/include/include/tarantool")
+
+    # Install tt.
+    env_cmd = [tt_cmd, "env"]
+    instance_process = subprocess.Popen(
+        env_cmd,
+        cwd=tmpdir,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+
+    # Check that the process shutdowned correctly.
+    instance_process_rc = instance_process.wait()
+    assert instance_process_rc == 0
+
```

Comment:
It would be nice that `tarantool` really is in path in this case, but it seems that CI installations mostly have tarantool installed from other source and this check would either be useless or malfunctioning.

Reply:
Yes, it will be pretty scuffed to test it changing real environment.